### PR TITLE
Upgrade mongo 3

### PR DESCRIPTION
**Pull request by aljones15**
This is a draft PR pending this Pr in [bedrock-passport](https://github.com/digitalbazaar/bedrock-passport/pull/41)

- Upgrades all the linter deps
- Upgrades all the test deps
- Adds working tests for all api endpoints
- Should be post-bedrock-identity ok

- [x] change bedrock-passport test dep from branch to next release

Addresses this issue: https://github.com/digitalbazaar/bedrock-zcap-storage-http/issues/3

### CODE REVIEW

File: test/mocha/mock-data.js [hunk 1]
```diff
@@ -3,9 +3,25 @@
...
 
+const zcap = {
+  id: 'urn:zcap:076a866b-1a9b-49d9-989b-e86757ad7fd2',
+  allowedAction: ['test'],
+  controller: 'did:key:foo',
+  '@context': 'did:content:foo',
+  parentCapability: 'urn:zcap:bar',
+  invocationTarget: 'urn:test:055283db-2c39-466a-a1b8-b0c12007b25e',
+  referenceId: 'did:test:foo',
+  // TODO is this necessary?
```
Comment (mattcollier) [inline test/mocha/mock-data.js:19]: @aljones15 can you please provide additional detail here?  I believe you were explaining that some API request fails without the `invoker` being specified along with `controller` which seems redundant to me.

Comment (aljones15) [reply test/mocha/mock-data.js:19]: Addressed here: https://github.com/digitalbazaar/bedrock-zcap-storage-http/pull/4/commits/f3ddda41204779d72aa3c168e697e7e2fdf4cc10

Comment (mattcollier) [reply test/mocha/mock-data.js:19]: @dlongley here's the code referenced in the TODO: https://github.com/digitalbazaar/bedrock-zcap-storage/blob/master/lib/storage.js#L240-L245

Does a ZCAP really need a controller AND an invoker?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
```
Comment (JSAssassin) [inline .github/workflows/main.yml:48]: Can we change the node versions in lint and test to 14.x?

File: test/mocha/mock-data.js (hunk 1 above)
Comment (mattcollier) [reply test/mocha/mock-data.js:19]: I've created an issuer for this as it does not need to hold up this PR

https://github.com/digitalbazaar/bedrock-zcap-storage-http/issues/5

Resolving this conversation.

**Review by mattcollier — APPROVED** [review]
Thanks for the tests!

**Pull request merged**